docs/cellxgene_census_docsite_data_release_info.md

```diff
@@ -1,23 +1,110 @@
 # Census data releases 
 
-**Last edited**: April, 2023.
+**Last edited**: July 6th, 2023.
+
+**Contents**
+
+1. [What is a Census data release?](#What-is-a-Census-data-release)
+2. [List of LTS Census data releases](#List-of-LTS-Census-data-releases)
 
 ## What is a Census data release?
 
 It is a Census build that is publicly hosted online. A Census build is 
```

Comment:
Might be worth stating:  Any given Census build is named with a unique tag, normally the date of build, e.g., `2020-01-30`